### PR TITLE
Various fixes from sonarqube

### PR DESCRIPTION
**Pull request by TrueTechy**

### CODE REVIEW

File: jellyfin_kodi/helper/api.py [hunk 1]
```diff
@@ -32,20 +32,14 @@ def get_playcount(self, played, playcount):
 
     def get_naming(self):
 
-        if self.item['Type'] == 'Episode':
+        if self.item['Type'] == 'Episode'and 'SeriesName' in self.item:
```
Comment (oddstr13) [inline jellyfin_kodi/helper/api.py:35]: Missing space around `and`

File: service.py [hunk 2]
```diff
@@ -25,7 +25,7 @@
 #################################################################################################
 
 LOG = LazyLogger(__name__)
-DELAY = int(settings('startupDelay') if settings('SyncInstallRunDone.bool') else 4 or 0)
+DELAY = int(settings('startupDelay') if settings('SyncInstallRunDone.bool') else 4)
```
Comment (oddstr13) [inline service.py:28]: Does `settings('startupDelay')` always return a number? This kinda looks like a fallback for None or empty string.

In any case, I'd move the `int()` to only wrap the setting
```suggestion
DELAY = int(settings('startupDelay')) if settings('SyncInstallRunDone.bool') else 4
```


```suggestion
DELAY = int(settings('startupDelay') or 0) if settings('SyncInstallRunDone.bool') else 4
```

File: jellyfin_kodi/helper/utils.py [hunk 3]
```diff
@@ -214,8 +214,8 @@ def set_screensaver(value):
 
 class JSONRPC(object):
 
-    version = 1
-    jsonrpc = "2.0"
+    id = 1
+    jsonrpc_version = "2.0"
```
Comment (oddstr13) [inline jellyfin_kodi/helper/utils.py:218]: I remember looking at this change myself, I do not remember why I didn't.

File: service.py (hunk 2 above)
Comment (TrueTechy) [reply service.py:28]: The startup delay is set in the addon settings and is limited to being a number

File: jellyfin_kodi/helper/api.py (hunk 1 above)
Comment (TrueTechy) [reply jellyfin_kodi/helper/api.py:35]: Damnit. Only looked through the code changes 3 times :/

File: jellyfin_kodi/helper/utils.py (hunk 3 above)
Comment (TrueTechy) [reply jellyfin_kodi/helper/utils.py:218]: Below is the only place I could find which referenced these variables and everything in my testing still worked so I'd assume it's all okay

**Comment by anthonylavado** [comment]
test

**Pull request merged**